routes/libraries.js

```diff
@@ -190,39 +208,38 @@ router.route('/:id')
     // Allows a user to delete their library
   });
 
-router.route('/:id/survey')
+router
+  .route("/:id/survey")
   .get(async (req, res) => {
     // Renders the survey page to rate fullness and input genres
-
-    let id;
-
     // If the library ID is not valid, render the error page with a status code of 400
+    let id;
     try {
       id = req.params.id;
       id = validation.checkValidId(id);
     } catch (e) {
-      return res.status(400).render('error', {errorCode: 400, searchValue: "Library"});
+      return res
+        .status(400)
+        .render("error", { errorCode: 400, searchValue: "Library" });
     }
-
     try {
-      res.render('libraries/fullness', { id: id });
+      res.render("libraries/fullness", { id: id });
```

Comment:
when you render a page it needs to have a title if it's a handlesbar page

Reply:
Added